File: BackBarbearia/Models/Atendimento.cs

using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Salao.Models
{
    public abstract class Atendimento
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id {get;set;}
        public int ServicoId{get;set;}
        public Servico? Servico{get;set;}
        public int ClienteId{get;set;}
        public  Cliente? Cliente{get;set;}
    }
}

File: FrontBarbearia/src/app/Atendimento.ts
import { Barbeiro } from "./Barbeiro";
import { Cliente } from "./Cliente";
import { Servico } from "./Servico";

export abstract class Atendimento {
    id: number = 0;
    servicoId: number= 0;
    servico: Servico| undefined;
    clienteId: number = 0;
    cliente: Cliente | undefined;
}